lib/elixir/lib/calendar.ex

```diff
@@ -70,6 +70,13 @@ defmodule Calendar do
   @callback leap_year?(year) :: boolean
 
   @doc """
+  Calculates the day of the week from the given `year`, `month`, and `day`.
+
+  Returns the day of the week as an integer, Monday being 1, Tuesday 2, and so on.
```

Comment:
Let's not mention Monday and so on here because it may not apply to all calendars.